includes/EDD_SL_Plugin_Updater.php

```diff
@@ -295,6 +295,13 @@ private function api_request( $_action, $_data ) {
             'url'        => home_url()
         );
 
+	$cache_key = md5( 'edd_plugin_' . sanitize_key(  $api_params['license'] . $this->version ) . '_' . $api_params['edd_action'] );
```

Comment:
Instead of having `edd_plugin_` in the cache key, I'd prefer we move it to outside of that, so when we have issues with transients not clearing (we've been seeing reports of this on some hosts) it's easier for us to identify EDD based transients.